kamon-core/src/main/scala/kamon/util/executors/ExecutorServiceMetrics.scala

```diff
@@ -0,0 +1,140 @@
+/*
+ * =========================================================================================
+ * Copyright © 2013-2015 the kamon project <http://kamon.io/>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ * =========================================================================================
+ */
+
+package kamon.util.executors
+
+import kamon.Kamon
+import kamon.metric.Entity
+import java.util.concurrent.{ ForkJoinPool ⇒ JavaForkJoinPool }
+import java.util.concurrent.{ ExecutorService, ThreadPoolExecutor }
+
+import scala.concurrent.forkjoin.ForkJoinPool
+import scala.util.control.NoStackTrace
+
+object ExecutorServiceMetrics {
```

Comment:
This is cool, but I'm not entirely sure about the fact that all the `register` methods are returning `Unit`. What about returning the entity recorder for the thread pool? that will make it consistent with the API exposed in Kamon.metrics.